### PR TITLE
[doc] modify install astra documentation for kinetic

### PR DESCRIPTION
**Pull request by yuki-shark**
- replace `indigo` to `$ROS_DISTRO`
- change workspace name from `ros/indigo`to `CATKIN_WS`

Cc @708yamaguchi

### CODE REVIEW

**Comment by 708yamaguchi** [comment]
Looks good to me.

File: doc/install_astra_camera.rst [hunk 1]
```diff
@@ -78,14 +78,16 @@ Then, edit ``/usr/lib/pkgconfig/libopenni2.pc`` to be like below::
 
 .. code-block:: bash
 
-  cd ~/ros/indigo/src
+  cd ~/CATKIN_WS/src
```
Comment (knorth55) [inline doc/install_astra_camera.rst]: why is it capital? `catkin_ws`?

Comment (708yamaguchi) [reply doc/install_astra_camera.rst]: I fixed the typo.

File: doc/install_astra_camera.rst [hunk 2]
```diff
@@ -78,7 +78,7 @@ Then, edit ``/usr/lib/pkgconfig/libopenni2.pc`` to be like below::
 
 .. code-block:: bash
 
-  cd ~/CATKIN_WS/src
+  cd ~/YOUR_CATKIN_WS/src
```
Comment (knorth55) [inline doc/install_astra_camera.rst]: How about `cd <your catkin workspace>/src`?

Comment (708yamaguchi) [reply doc/install_astra_camera.rst]: I think it's good.
I rebased our previous commits.

**Pull request merged**